### DISCUSSION TITLE
Multiselect Button Customization AND/OR prevent Dropdown Menu from being closed on Menu Checkbox Item

### DISCUSSION THREAD

**Discussion by dajoeberlei** [livewire/flux#681] [Feature Requests]
hi there,

what I'm trying to accomplish are two buttons that customize a table view. 
1. a filter Button. The button gets colored / active / has an icon instead of a text
2. a dropdown menu/multi select in which i can select the table cells that should be shown

I both tried dropdown menu and multiselect. But I cant make the trigger button of the multi select look how i want to. I also tried dropdown menu but this one closes everytime I click one of the <flux:menu.checkbox ... > items. I need it to stay open when I select a table column option or a filter.

This is what im trying to accomplish:
<img width="591" alt="grafik" src="https://github.com/user-attachments/assets/65f61563-9ae2-4065-b249-c170a38997e7">

**Comment by joshhanley** [comment]
Closing this as it is a duplicate request. Follow #130 for more details.
